docs/recipes/gulp/gulpfile.js

```diff
@@ -2,34 +2,57 @@
 
 const gulp = require('gulp');
 const connect = require('gulp-connect');
-const lighthouse = require('../../../lighthouse-core');
-const perfConfig = require('../../../lighthouse-core/config/perf.json');
-const port = 8080;
+const lighthouse = require('lighthouse');
+const ChromeLauncher = require('lighthouse/lighthouse-cli/chrome-launcher');
+const perfConfig = require('lighthouse/lighthouse-core/config/perf.json');
+const PORT = 8080;
+let launcher;
 
 const connectServer = function() {
   return connect.server({
     root: '../public',
     livereload: true,
-    port: port
+    PORT
   });
 };
 
-function handleOk() {
+const disconnectServer = function() {
   connect.serverClose();
+  launcher.kill();
+};
+
+const launchChrome = function() {
+  launcher = new (ChromeLauncher.ChromeLauncher || ChromeLauncher)();
```

Comment:
I dunno, it was just dumb copy/paste from https://github.com/paulirish/pwmetrics/blob/master/lib/index.js#L122
maybe @paulirish knows more...